app/src/main/java/org/fossasia/susi/ai/rest/responses/others/SpeakerConfigResponse.kt

```diff
@@ -0,0 +1,14 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+class SpeakerConfigResponse {
+
```

Comment:
Can this be converted to data class?

Reply:
I think yes, but what difference does it make , all the response classes were made like this so made this the same way as well

Reply:
Well, we should not make the mistakes previous developers made. Plus those classes were converted from Java to Kotlin, which does not automatically change them to data classes

`but what difference does it make`
Also, read up why data classes are used and what's the difference from normal classes, you'll know why I'm telling you to use data classes